inputs/orchestrator_inner:2.json

```diff
@@ -0,0 +1,55 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "hub": "{{KOJI_HUB}}",
+        "repos": []
+      }
+    },
```

Comment:
@twaugh we had to remove `add_labels_from_dockerfile` plugin from here because the base image is not inspected in this case (filesystem tarball is not streamed into docker engine of orchestrator build).

We can either keep this plugin out of here, or change the plugin [here](https://github.com/projectatomic/atomic-reactor/blob/master/atomic_reactor/plugins/pre_add_labels_in_df.py#L279) to fallback to an empty Labels dict if base image is not inspected.

I think the fallback mechanism makes sense. What do you think?